prime-dotnet-webapi-tests/Unit Tests/RazorConverterService.Tests.cs

```diff
@@ -142,17 +234,16 @@ public async void TestRender_SiteSummary()
 
         [Theory]
         [MemberData(nameof(DocumentTemplates))]
-        public async void TestRender_Documents(RazorTemplate<Document> template)
+        public async void TestRender_Documents(RazorTemplate<File> template)
         {
             var service = CreateService();
-            var model = new Document("filename.ext", new byte[] { });
+            var model = new File("filename.ext", new byte[] { });
 
             var html = await service.RenderTemplateToStringAsync(template, model);
 
             Assert.NotNull(html);
         }
```

Comment:
Would it be better to split apart this test, so that `DocumentTemplates` data is not used?  Then there can be a more substantial assertion than simply `Assert.NotNull(html);`?